plasma/utils/utils.py

```diff
@@ -21,7 +21,7 @@ def bytes_fill_left(inp, length):
 
 
 def confirm_tx(tx, root, key):
-    return sign(u.sha3(tx.hash + tx.sig1 + tx.sig2 + root), key)
+    return sign(u.sha3(tx.hash + root), key)
```

Comment:
@DavidKnott why input sigs don't need to be included in confirmation anymore?

Reply:
The tx `sender` needs to give up custody of their funds after seeing that their transaction has been included on Ethereum, there's no need for them to sign the tx signatures as well.